src/public/sass/certificates.scss

```
@@ -0,0 +1,70 @@
+@use 'base/variables' as v;
+@use 'base/mixins' as m;
+
+.certificates {
+    &__heading {
+        margin-top: 5rem;
+        @include m.heading;
+        @include m.fluid-text(3rem, 6rem);
+    }
+
+    &__grid {
+        max-width: 90%;
+        margin: 5rem auto 0 auto;
+        @include m.grid(1, 1.5rem);
+        @include m.phoneXL{
+            @include m.grid(2, 4rem 2rem);
+        }
+        @include m.tablet{
+            row-gap: 3rem;
+            padding: 2rem 2rem 0 2rem;
+        }
+        @include m.desktop {
+            @include m.flex-columns(3, 2rem, 4rem);
+            
+        }
+    }
+}
+
+.certificate {
+    padding: 2rem 2rem 0 2rem;
+    border: 1px solid v.$aqua-blue;
+    transition: border 100ms ease-in-out;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
```

```suggestion
    justify-content: space-between;
    box-sizing: border-box;
    transition: box-shadow 500ms ease-in-out;
```
check the next comment